README.md

```diff
@@ -43,35 +43,86 @@ It will create some environment variables to ease access to some folders:
 ```bash
 source WRemnants/setup.sh
 ```
+### Theory agnostic analysis
 
+Make histograms (only nominal and mass variations for now, systematics are being developed)
+```
+/usr/bin/time -v python scripts/histmakers/mw_with_mu_eta_pt.py -o outputFolder/ --met DeepMETReso  --addHelicityHistos --pt 30 26.0 56.0 --onlyMainHistograms
```

Comment:
Should we not make DeepMETReso and the pt range the default?

Reply:
Yeah, at some point deepMET will be default, but at the moment it is not because we don't have the uncertainties (but for the theory agnostic we would not be using them for the moment in any case). 
This pt range could also become the new default.

Essentially here I just wanted to share the commands I had recently used, rather than changing the default values and potentially modify everyone's settings.

Reply:
Ok, but is the purpose of the readme to instruct people totally unfamiliar with the code, or to remind yourself what commands you use? If it's the former, the difference between deepMet or not should make no difference. I guess the later is also ok but it's maybe not optimal